templates/migrate/phobos_03_add_created_at_to_events.rb.erb

```diff
@@ -0,0 +1,9 @@
+class Phobos03AddCreatedAtToEvents < ActiveRecord::Migration[<%= ActiveRecord::VERSION::STRING.to_f %>]
```

Comment:
Inspired by the read https://devcenter.heroku.com/articles/postgresql-indexes#b-trees-and-sorting indices appropriate for this task have been added.